src/app/functions/server/getExperimentationId.ts

```diff
@@ -28,6 +28,12 @@ export function getExperimentationId(
     // If the user is logged in, use the Subscriber ID.
     const namespace = process.env.NIMBUS_UUID_NAMESPACE;
     if (!namespace) {
+      logger.error(
+        "NIMBUS_UUID_NAMESPACE environment variable is missing. Cannot generate experimentationId.",
+        {
+          accountId,
```

Comment:
Let's avoid logging the `accountId`, any problem here is unlikely to be related to the user (more likely the variable we need is not set)